viral/utils.py

```
@@ -6,6 +6,8 @@
 from matplotlib import pyplot as plt
 import numpy as np
 from enum import Enum
+import pandas as pd
+from scipy.stats import zscore
```

Ruff (0.8.2)

`scipy.stats.zscore` imported but unused

Remove unused import: `scipy.stats.zscore`

(F401)